packages/manager/apps/web/client/app/hosting/multisite/git-integration/deployment/git-deployment.routing.js

```diff
@@ -0,0 +1,38 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.dashboard.multisite.git-deployment', {
+    url: '/git-deployment?path',
+    params: {
+      path: null,
+    },
+    layout: 'modal',
+    views: {
+      modal: {
+        component: 'hostingMultisiteGitDeploymentComponent',
+      },
+    },
+    resolve: {
+      goBack: /* @ngInject */ ($transition$, $state, $timeout, Alerter) => (
+        message = false,
+        type = 'info',
+      ) => {
+        const reload = message && type === 'info';
+        const promise = $state.go('app.hosting.dashboard.multisite', null, {
+          reload,
+        });
+        if (message) {
+          promise.then(() =>
+            $timeout(() =>
+              Alerter.set(`alert-${type}`, message, null, 'app.alerts.tabs'),
+            ),
+          );
+        }
+        return promise;
+      },
```

Comment:
**question** : it's already defined in `packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.routing.js`. Is there any possibility to group it ?

Reply:
It's fixed